template/module/README.rst

```diff
@@ -37,6 +37,14 @@ Known issues / Roadmap
 
 * ...
 
+Bug Tracker
+===========
+
+Bugs are tracked on GitHub.
+In case of trouble, please check `there <https://github.com/OCA/{project_repo}/issues>`_ if your issue has already been reported.
```

Comment:
Suggestion:

    Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
    In case of trouble, please check there if your issue has already been reported.